Tests/Common++Test/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ include(FetchContent)
 FetchContent_Declare(
     googletest
     GIT_REPOSITORY https://github.com/google/googletest.git
-    GIT_TAG release-1.11.0
+    GIT_TAG release-1.12.0
```

Comment:
I don't know that we already have gtest.

Reply:
We didn't the fetch content will pull it from internet

Reply:
I know. I meant I am suprised that there is gtest fetching in current master because we are not using gtest now.

Reply:
> I am suprised that there is gtest fetching in current master

Wdym, @tigercosmos ? There isn't one in the master branch?

Reply:
@tigercosmos you're looking at a specific commit and not the whole MR